tools/api/validate_structure.py

```diff
@@ -19,7 +19,7 @@
 
 # These are trees that allow v3+ protos, but only a strict whitelist.
```

Comment:
What is this needed for now @htuch?

Reply:
I think this is legacy from the `v3alpha` to `v3` move, I will remove.

Reply:
https://github.com/envoyproxy/envoy/pull/11145